generators/app/index.js

```diff
@@ -284,8 +284,12 @@ module.exports = class extends BaseGenerator {
                 }
             });
 
-            const kafkaProperties = jsYaml.dump(kafkaPreviousConfiguration, { lineWidth: -1, sortKeys: false });
-            const kafkaTestProperties = jsYaml.dump(kafkaPreviousTestConfiguration, { lineWidth: -1, sortKeys: false });
+            const kafkaProperties = jsYaml.dump(utils.orderRootPropertiesAsFirst(kafkaPreviousConfiguration), {
+                lineWidth: -1
+            });
+            const kafkaTestProperties = jsYaml.dump(utils.orderRootPropertiesAsFirst(kafkaPreviousTestConfiguration), {
+                lineWidth: -1
+            });
```

Comment:
Maybe rename the new function simplier?

```suggestion
            const kafkaProperties = jsYaml.dump(utils.sortKafkaProperties(kafkaPreviousConfiguration), {
                lineWidth: -1
            });
            const kafkaTestProperties = jsYaml.dump(utils.sortKafkaProperties(kafkaPreviousTestConfiguration), {
                lineWidth: -1
            });
```